src/module/Ui/templates/entity/view/course-page.twig

```diff
@@ -22,7 +22,7 @@
 
 <div class="row">
     <div class="col-xs-12">
-        <h2 class="heading-content">{{ revision.get('title') }}</h2>
+        <h1 class="heading-content">{{ revision.get('title') }}</h1>
```

Comment:
@arekkas Yes, previously (when we still had the bootstrap carousel) there was the title of the course as h1, however in the current design only the title of the coursepage is displayed and therefore should be h1.